mmv1/third_party/terraform/services/container/resource_container_cluster.go.erb

```diff
@@ -6147,6 +6145,21 @@ func containerClusterAutopilotCustomizeDiff(_ context.Context, d *schema.Resourc
 	return nil
 }
 
+// Default to VPC_NATIVE mode during initial creation
+func containerClusterNetworkingModeCustomizeDiff(_ context.Context, d *schema.ResourceDiff, meta interface{}) error {
+	// skip if not during initial creation
+	o, _ := d.GetChange("name")
+	if o != "" {
+		return nil
+	}
+
+	err := d.SetNew("networking_mode", "VPC_NATIVE")
```

Comment:
Can you comment on the field itself to indicate we supply a default through a CD?